recipe/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
     - pathlib  # [py27]
     - ujson >=1.35
     - dill >=0.2,<0.3
-    - regex >=2017.4.5,<2017.12.1
+    - regex =2018.01.10
```

Comment:
Thanks for your help with this feedstock!

I'm not sure about this pin though. We need to keep the regex pin aligned with the pip version of spaCy. Is there a reason you want to pin to this version specifically? I know the latest versions mess things up (although I haven't gotten to the bottom of why...)

Reply:
This pin is actually pulled from the git tag for the spacy release the meta.yml here was adopted. So this should be the correct version.